### PR TITLE
Fix cruft file remover information retrieval

### PR DESCRIPTION
**Pull request by mikkoaf**
This commit fixes the issue #153 by removing sub-directories and files from the listings. This also changes how the cruft remover shows file size information. Previously the shown directory size information was about the first directory within the directory, if applicable.

### CODE REVIEW

File: lib/cruftfiles-ajax.php [hunk 1]
```diff
@@ -6,10 +6,11 @@
...
   exec( 'du ' . $file . ' -h --time', $output );
-  $size = explode("\t", $output[0]);
-
-  $data['size'] = $size[0];
-  $data['mod_date'] = $size[1];
+  $size = explode("\t", end($output));
+  $mod_date = array_values(array_slice($size, -2, 1 ))[0];
+  $size = array_values(array_slice($size, -3, 1 ))[0];
+  $data['size'] = $size;
+  $data['mod_date'] = $mod_date;
```
Comment (l3ku) [inline lib/cruftfiles-ajax.php]: Could you please explain why this change is done? I tried this in PHP interactive and couldn't find any scenario in which the already existing code would not work:
```
php > exec('du /data/wordpress/nginx/ -h --time', $output);
php > $size = explode("\t", end($output));
php > print_r($size);
Array
(
    [0] => 12K
    [1] => 2018-07-11 15:07
    [2] => /data/wordpress/nginx/
)
php > exec('du /data/wordpress/ -h --time', $output);
php > $size = explode("\t", end($output));
php > print_r($size);
Array
(
    [0] => 120M
    [1] => 2018-08-06 10:39
    [2] => /data/wordpress/
)
php > exec('du /data/wordpress/* -h --time', $output);
php > $size = explode("\t", end($output));
php > print_r($size);
Array
(
    [0] => 652K
    [1] => 2017-08-10 19:09
    [2] => /data/wordpress/vendor
)
php > exec('du /data/wordpress/htdocs/wp-content/mu-plugins -h --time', $output);
php > $size = explode("\t", end($output));
php > print_r($size);
Array
(
    [0] => 6.9M
    [1] => 2018-08-06 10:39
    [2] => /data/wordpress/htdocs/wp-content/mu-plugins
)
```
The above would in my opinion imply that the file size would always be in $size[0], the modification date in $size[1] and filename in $size[2].

Comment (mikkoaf) [reply lib/cruftfiles-ajax.php]: During testing and developing I recall running into a situation where this change was solved a problem I ran into.
However, I can not reproduce root cause for this change at this point in time.

**Comment by mikkoaf** [comment]
Rebased and addressed the requested changes
